test/e2e/test_cluster_configs/windows/cse_benchmark_win_vhds.json

```diff
@@ -4,8 +4,6 @@
 	},
 	"options": {
 		"allowedOrchestratorVersions": ["latestReleasedVersion"],
-		"clientId": "AKS_ENGINE_e7b1dca4_CLIENT_ID",
```

Comment:
These currently run in the same jenkins pipeline (the 1.16 matrix i believe) so we'll need to move them into two seperate pipelines.